### PR TITLE
Adds the documentation roadmap

### LINKED ISSUE #1: Indicating areas of documentation open to contributions
Are there any areas of this documentation project that are open to contributions? I'd be happy to help writing docs, but don't want to accidentally duplicate effort. If contributions would be useful, would you be willing to raise #good-first-issue tickets for these, so I (or others) could pick them up?

### PR DESCRIPTION
**Pull request by aivascu**
Adds a documentation roadmap to the readme file for a more transparent documentation process.

resolves #1

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -12,3 +12,33 @@ There are a couple of ways to setup the local development environment:
...
+- [ ] Freeze and Inject features
+- [ ] Testing frameworks integration
+  - [ ] xUnit
+  - [ ] NUnit
+- [ ] Mocking framework integration
+  - [ ] Moq
+  - [ ] NSubstitute
+  - [ ] FakeItEasy
+  - [ ] Foq
+  - [ ] RhinoMock
```
Comment (zvirja) [inline readme.md]: I would not describe this, as RhinoMock is not being developed anymore. Actually, I would just drop support of it in v5 if you ask me :)

File: readme.md [hunk 2]
```diff
@@ -12,3 +12,33 @@ There are a couple of ways to setup the local development environment:
...
+- [ ] Idioms
+  - [ ] Assertions
+  - [ ] FsCheck
+- [ ] Internal library design and philosophy
```
Comment (zvirja) [inline readme.md:39]: Add `Behaviors` here as well. They are a sort of customizations, but a bit different and complicated.

**Review by zvirja — APPROVED** [review]
LGTM 👍  Left a couple of minor comments.

File: readme.md (hunk 1 above)
Comment (aivascu) [reply readme.md]: agreed )

File: readme.md (hunk 2 above)
Comment (aivascu) [reply readme.md:39]: done

**Pull request merged**